local-tests/test_update.ipynb

```diff
@@ -207,6 +211,17 @@
     "else:\n",
     "    print(\"DIFFERENT!\")"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9ede429e-fca2-41fe-80c7-ac7df1606ddc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if oldver == newver:\n",
+    "    print(\"ERROR: runtime version reported by nodes didn't change after the update\")"
+   ]
```

Comment:
missing `sys.exit(1)`; also in the previous lines (~61 the notebook has changes that are not included in the script itself)

Reply:
Yes, on purpose. Notebook is not supposed to do things like sleep, sys.exit etc. It's for humans to use :)